openff/toolkit/tests/test_molecule.py

```diff
@@ -3332,8 +3215,8 @@ def test_trivalent_lone_pair_virtual_site_position(self):
         atom3 = molecule.atoms[2]
         atom4 = molecule.atoms[3]
         distance = 1.0 * unit.angstrom
-        vsite1_index = molecule.add_trivalent_lone_pair_virtual_site(
-            [atom1, atom2, atom3, atom4], distance, symmetric=False
+        molecule.add_trivalent_lone_pair_virtual_site(
+            [atom2, atom1, atom3, atom4], distance
```

Comment:
Trivalent atom ordering now matches SMIRNOFF spec, so need to update the tests which still use the incorrect order